contrib/net/http/option.go

```diff
@@ -168,6 +176,14 @@ func RTWithResourceNamer(namer func(req *http.Request) string) RoundTripperOptio
 	}
 }
 
+// RTWithOperationNamer specifies a function which will be used to
+// obtain the span operation name for a given request.
+func RTWithOperationNamer(namer func(req *http.Request) string) RoundTripperOption {
```

Comment:
This change expands our public interface, we should double check we want to do this here (or if there's any other examples in other `contrib` packages where we already offer this or a similar config option)

Reply:
As discussed offline, we will add this options with the name `RTWithSpanNamer`